### ISSUE TITLE
Add Telemetry SDK's and Flex to Create Integrations section

### ISSUE THREAD

**Issue by danifitz** [newrelic/docs-website#439]
### Is your feature request related to a problem? Please describe

There is a create integrations section at the top level nav. When I click on it, I'd expect to find information about our wonderful Telemetry SDK's and Flex which enable customers to create integrations. https://docs-preview.newrelic.com/docs/create-integrations

### Describe the solution you'd like

Links to Telemetry SDK's docs and Flex docs from the Create an integration page.

### Describe why this important to you

As a customer, to discover the full potential of integration options, I'd like to be able to discover the Telemetry SDK's or Flex as options for creating integrations, since they are both very powerful solutions.

### Additional context

None.

**Comment by jerelmiller** [comment]
Thanks so much for taking a look!

Please see [this comment](https://github.com/newrelic/docs-website/issues/438#issuecomment-748255111) in regards to the issue. Its likely that we haven't migrated this content over yet 😄

**Comment by zuluecho9** [comment]
A few notes on this: 

• This section was previously hidden/unindexed but don't think we can do that on github site.
• The docs are currently out of order; the first doc should be the 'Intro to Integrations SDK', which does include some info about other ways to create integrations. 
• Maybe we just rename this category from 'Create integrations' to 'Infra integrations SDK' or something more descriptive, less general.

**Comment by jerelmiller** [comment]
Hey @zuluecho9! We are on our final polish step for the site and will be tackling these issues in our current sprint! We've got these issues captured between a few different ones:

> This section was previously hidden/unindexed but don't think we can do that on github site.

We've created a private docs site that will now display all hidden pages. All of the current private docs content is migrated there, so this should already be done (work done as part of #605)

> The docs are currently out of order; the first doc should be the 'Intro to Integrations SDK', which does include some info about other ways to create integrations.

The sort order issue has been noted and we are planning to fix as a part of #622.

> Maybe we just rename this category from 'Create integrations' to 'Infra integrations SDK' or something more descriptive, less general.

We've also noted the `Create integrations` section should not be there. We are planning to tackle that as part of #587 and move those docs to the proper location.
